buildrunner/__init__.py

```diff
@@ -65,6 +65,7 @@
 
 DEFAULT_CACHES_ROOT = '~/.buildrunner/caches'
 DEFAULT_RUN_CONFIG_FILES = ['buildrunner.yaml', 'gauntlet.yaml']
+DEFAULT_BUILD_SINGLE_PLATFORM_OVERRIDE = False
```

Comment:
This global setting seems a little overkill for a `False` :)

Reply:
What are you talking about? That isn't there... :)